alert-database/src/main/java/com/synopsys/integration/alert/database/job/jira/cloud/JiraCloudJobDetailsAccessor.java

```diff
@@ -68,6 +67,8 @@ public JiraCloudJobDetailsEntity saveJiraCloudJobDetails(UUID jobId, JiraCloudJo
         if (!customFieldsToSave.isEmpty()) {
             List<JiraCloudJobCustomFieldEntity> savedCustomFields = jiraCloudJobCustomFieldRepository.saveAll(customFieldsToSave);
             savedJobDetails.setJobCustomFields(savedCustomFields);
+        } else {
+            savedJobDetails.setJobCustomFields(List.of());
```

Comment:
Oh wow. This is probably a bug in a few other places as well. I'll create a ticket against 6.4.0